shared/test/test_redis_table.rb

```diff
@@ -153,8 +161,8 @@ def test_delete_many
 
   def test_expiration
     # Test with a 1-second expire time
-    expire_time = 2
-    margin_time = 0.2
+    expire_time = 0.01
+    margin_time = 0.002
```

Comment:
Correct.  I was able to run this file 100 times in a row with zero failures at these small timeouts, which I considered a sufficiently low flakiness rate.  If timecop will play nice with fakeredis though, I'll switch to that.